terraform-modules/cloudsql-postgres/cloudsql.tf

```diff
@@ -57,6 +57,11 @@ resource "google_sql_database_instance" "cloudsql_instance" {
     }
 
     user_labels = var.cloudsql_instance_labels
+
+    database_flags {
```

Comment:
I am not sure that setting the Postgres parameters actually affects the number of connections, unless it is used in some way to scale the size of the CloudSQL instance we are deploying.

We had the Postgres max_connections configuration set to something like 1000 when we ran out with 50.

I may be wrong about this, of course...

Reply:
https://cloud.google.com/sql/docs/quotas#postgresql indicates that this value can be overridden, but the feature is in beta.